interfaces/builtin/serial_port.go

```diff
@@ -154,7 +154,8 @@ func (iface *SerialPortInterface) ConnectedPlugSnippet(plug *interfaces.Plug, sl
 	switch securitySystem {
 	case interfaces.SecurityAppArmor:
 		if iface.hasUsbAttrs(slot) {
-			// This apparmor rule must match serialDeviceNodePattern
+			// This apparmor rule is an approximation of serialDeviceNodePattern
+			// specifically AARE does not support the '+' token for "one or more"
```

Comment:
Sounds fine. We'll never even ask AA to mediate if we don't get a proper path that passes the primary/richer Go one.